api/src/paths/projects.inttest.ts

```diff
@@ -0,0 +1,60 @@
+export {};
```

Comment:
The filename will need to be `*.test.ts` or it won't get picked up by the test runner

Reply:
I changed that intentionally in other for the integration test to not run ... as I want to rule out the integration test as the issue ... let's chat.